pages/common/zip.md

```diff
@@ -8,7 +8,7 @@
 
 - E[x]clude unwanted files from being added to the compressed archive:
 
-`zip -r {{compressed.zip}} {{path/to/dir}} -x \*.git\* \*node_modules\* ...`
+`zip -r {{compressed.zip}} {{path/to/dir}} -x "{{.*}}" "{{node_modules}}" ...`
```

Comment:
I think we should keep the ".git". It's common enough to warrant a mention here.

Reply:
I would normally disagree, since hidden files of any kind are, by definition, more common than the `.git` directory. However, in this case I'm afraid the wildcard expression can be confusing, since that same combination (`.*`) has a specific meaning as a regex pattern.